lib/runners/processor/haml_lint.rb

```diff
@@ -66,48 +76,42 @@ def setup
     end
 
     def analyze(_changes)
-      options = [include_linter, exclude_linter, exclude, haml_lint_config].compact
-      run_analyzer(file, options)
+      run_analyzer
     end
 
+    private
+
     def setup_default_config
-      # NOTE: We expect default_rubocop.yml to be located in $HOME.
-      default_config = (Pathname(Dir.home) / './default_rubocop.yml').realpath
-      path = Pathname("#{current_dir}/.rubocop.yml")
-      return false if path.exist?
-      path.parent.mkpath
-      FileUtils.cp(default_config, path)
-      true
+      config_file = ".rubocop.yml"
+      return if File.exist? config_file
```

Comment:
This change is excellent :+1: In addition to this, I think the presence of `haml_lint_config` should be checked before copying the default config in case the custom RuboCop configuration requires an older version of Meowcop.

```suggestion
      return if File.exist? config_file
      return unless haml_lint_config.empty?
```

Reply:
Good catch! 👍

Reply:
Umm..., when I think again, this `setup_default_config` method is for `.rubocop.yml`, not for [`.haml-lint.yml`](https://github.com/sds/haml-lint#configuration).

So, it seems to me that we should just rename `setup_default_config` to `setup_rubocop_default_config` and keep this method's logic.
What do you think?

Reply:
OK, I misunderstood the configuration is for RuboCop, but it is for HAML-Lint. Please just rename it.

Reply:
Thanks for the feedback! I've fixed it via fcdb1b62.